README.md

```diff
@@ -8,22 +8,24 @@ If you have any issues using the API or would like some help, please open [an is
 
 ## API Overview
 
-The API tends to be RESTful and returns results in JSON by default.
+The API is generally RESTful and returns results in JSON by default.
 This means for example that it will return a 404 HTTP status code if a record can't be found.
 
-In general, any page you get through the website has a corresponding representation in the API obtained by prefixing its URL with `/api/`.
+In general, most pages you get through the website have a corresponding representation in the API obtained by prefixing the path component of the URL with `/api/`.
 For example, the data displayed at
 ```
-https://inspirehep.net/literature?sort=mostrecent&size=25&page=1&q=title%20api
+https://inspirehep.net/literature?sort=mostrecent&size=25&page=1&q=title api
 ```
 is available through the API at
 ```
-https://inspirehep.net/api/literature?sort=mostrecent&size=25&page=1&q=title%20api
+https://inspirehep.net/api/literature?sort=mostrecent&size=25&page=1&q=title api
 ```
 
+Currently only read-only operations are allowed and they all use the `GET` HTTP method.
```

Comment:
@tsgit commented:
>I would also emphasize that it currently is strictly a one way data retrieval
-- usually GET method, maybe POST for a query is also supported ?
>
>there is no support for data corrections via "POST" like a record "update" or
"deletion" or is there?
>
>are there plans to allow for authenticated (oauth2) changes via POST to
conferences, jobs, seminars, etc. ? that could be really  useful